lib/masamune/filesystem.rb

```diff
@@ -88,7 +87,7 @@ def relative_path?(path)
       path[0] != '/'
     end
 
-    def parent_paths(path, &block)
+    def parent_paths(path)
```

Comment:
it's not really necessary to enumerate the parent paths since this should be small. no caller of this method was using block form